extensions/amp-date-countdown/0.1/amp-date-countdown.js

```diff
@@ -154,6 +157,9 @@ export class AmpDateCountdown extends AMP.BaseElement {
     /** @private {!Object|null} */
     this.localeWordList_ = this.getLocaleWord_(this.locale_);
 
+    /** @private {boolean} */
+    this.countUp_ = this.element.hasAttribute('count-up');
```

Comment:
Let's do `data-`, and remove validator changes:

```suggestion
    this.countUp_ = this.element.dataset['countUp'];
```

Reply:
Awesome, updated in the latest revision.  Stuck with `hasAttribute` since I wanted a boolean return value.